SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/TypeStructure.cpp

```diff
@@ -79,6 +79,21 @@ uint32 GenerateChecksum(GDK_PROPERTY(Property) * Property, uint32 ParentChecksum
 	Checksum = FCrc::StrCrc32(*Property->GetCPPType(nullptr, 0).ToLower(), Checksum); // Evolve by property type
 	Checksum = FCrc::MemCrc32(&StaticArrayIndex, sizeof(StaticArrayIndex),
 							  Checksum); // Evolve by StaticArrayIndex (to make all unrolled static array elements unique)
+#if ENGINE_MINOR_VERSION >= 27 || ENGINE_MAJOR_VERSION == 5
```

Comment:
```suggestion
#if UE_VERSION_NEWER_THAN(4,26,0)
```
Found a nicer way of doing this ^.

Reply:
nice, unfortunate that it needs an extra include, they could have just put it next to the engine version definitions, I think, but I switched to using it. (Also needed 4.27.-1, which is a bit annoying, but ok)